test/torchaudio_unittest/prototype/hdemucs_test_impl.py

```diff
@@ -0,0 +1,105 @@
+import torch
+from parameterized import parameterized
+from torchaudio.prototype.models.hdemucs import HDemucs, _HEncLayer, _HDecLayer
+from torchaudio_unittest.common_utils import TestBaseMixin, torch_script
+
+_SOURCE_SETS = [
+    (["bass", "drums", "other", "vocals"],),
+    (["bass", "drums", "other"],),
+    (["bass", "vocals"],),
+    (["vocals"],),
+]
+
+
+def _get_hdemucs_model(sources):
+    return HDemucs(sources)
+
+
+class TorchscriptConsistencyMixin(TestBaseMixin):
```

Comment:
let's remove the torchscript tests for this PR. we can follow up on torchscriptability in the future

Reply:
Are we keeping this?